science/udunits2/files/patch-_XOPEN_SOURCE.diff

```diff
@@ -0,0 +1,58 @@
+Need snprintf(), strdup()
+See: https://trac.macports.org/ticket/61032
+
+--- lib/converter.c.orig
++++ lib/converter.c
+@@ -12,7 +12,7 @@
+ /*LINTLIBRARY*/
+ 
+ #ifndef	_XOPEN_SOURCE
+-#   define _XOPEN_SOURCE 500
++#   define _XOPEN_SOURCE 600
+ #endif
+ 
+ #ifdef _MSC_VER
+--- lib/formatter.c.orig
++++ lib/formatter.c
+@@ -11,7 +11,7 @@
+ /*LINTLIBRARY*/
+ 
+ #ifndef	_XOPEN_SOURCE
+-#   define _XOPEN_SOURCE 500
++#   define _XOPEN_SOURCE 600
+ #endif
+ 
+ #ifdef _MSC_VER
+--- lib/parser.c.orig
++++ lib/parser.c
+@@ -89,7 +89,7 @@
+ /*LINTLIBRARY*/
+ 
+ #ifndef	_XOPEN_SOURCE
+-#   define _XOPEN_SOURCE 500
++#   define _XOPEN_SOURCE 600
+ #endif
+ 
+ #include <assert.h>
+--- lib/unitAndId.c.orig
++++ lib/unitAndId.c
+@@ -12,7 +12,7 @@
+ /*LINTLIBRARY*/
+ 
+ #ifndef	_XOPEN_SOURCE
+-#   define _XOPEN_SOURCE 500
++#   define _XOPEN_SOURCE 600
+ #endif
+ 
+ #include <assert.h>
+--- lib/xml.c.orig
++++ lib/xml.c
+@@ -15,7 +15,7 @@
+ #include <config.h>
+ 
+ #ifndef	_XOPEN_SOURCE
+-#   define _XOPEN_SOURCE 500
++#   define _XOPEN_SOURCE 500
```

Comment:
No change here?

Reply:
Fixed, thanks for catching this. (Something that can only happen when making a patch by hand, as one should avoid doing.)